JustSaying.Messaging/MessageProcessingStrategies/Throttled.cs

```diff
@@ -65,7 +65,7 @@ public int FreeTasks
 
         public void ProcessMessage(Func<Task> action)
         {
-            var task = new Task<Task>(async () => await action());
+            var task = new Task<Task>(action);

```

Comment:
Hm, I thought it was for the Threadpool scheduling, but would it work without that? So long as there is still the `task.Start()` after?